src/app/community/community.component.ts

```diff
@@ -29,7 +30,8 @@ export class CommunityComponent implements OnInit, AfterViewInit {
 
   constructor(
     private couchService: CouchService,
-    private dialog: MatDialog
+    private dialog: MatDialog,
+    private route: ActivatedRoute
   ) {}
 
   ngAfterViewInit() {
```

Comment:
Do we need this. There is no code using.

Reply:
there is a code but its not of this commit so its not showing.